policy-engine/src/main.rs

```diff
@@ -80,6 +80,7 @@ fn main() -> Result<(), Error> {
     registry.register(Box::new(BUILD_INFO.clone()))?;
     HttpServer::new(move || {
         App::new()
+            .wrap(Logger::default())
```

Comment:
```suggestion
            .wrap(actix_web::middleware::Logger::default())
```

Reply:
This suggestion will make it compile but it won't affect the log messages we produce manually, only the ones generated internally in the actix_web components.

Reply:
Pushed the fix with efdfd8e -> ea5c4e8.

> This suggestion will make it compile but it won't affect the log messages we produce manually...

Is there a way to add the user-agent prefix to *all* the log messages we produce manually while handling a request?  I'd rather not instrument each of our log lines directly like #186.

Reply:
> Is there a way to add the user-agent prefix to all the log messages we produce manually...

Sounds like #186 [does this][1], although I'm not clear on the mechanism.

/close

[1]: https://github.com/openshift/cincinnati/pull/186#discussion_r355347588